pipeline/src/node_properties/model_inference_properties.rs

```diff
@@ -18,9 +18,10 @@ pub struct ModelInferenceProperties {
     #[serde(flatten)]
     pub runtime: Option<ModelInferenceRuntime>,
 
-    /// Vector containing the min inference threshold value for each class.
+    /// Maps each class label (key) to the ClassInferenceProperties set.
+    /// Use "*" as key to specify global properties that should affect all the model classes.
     #[serde(default)]
```

Comment:
I wonder whether we should also add `skip_serializing_if = "BTreeMap::is_empty"` to the serde attributes.